src/DotNetWorker.Core/Context/Features/DefaultFunctionInputBindingFeature.cs

```diff
@@ -97,6 +100,21 @@ public async ValueTask<FunctionInputBindingResult> BindFunctionInputAsync(Functi
                         errors.Add(
                             $"Cannot convert input parameter '{param.Name}' to type '{param.Type.FullName}' from type '{source.GetType().FullName}'. Error:{bindingResult.Error}");
                     }
+                    else if (bindingResult.Status == ConversionStatus.Unhandled)
+                    {
+                        // If still unhandled after going through all converters,check an explicit default value was provided for the parameter.
+                        if (param.DefaultValue is not null)
```

Comment:
This looks like it won't support `null` default values, ie: `MyFunction(string str = null)`. Is this intentional?

Reply:
Yea, the original plan was to throw an error in that case. But I realize it will be confusing, I pushed a new iteration which handles this case. So if you have "string foo = null" , it will be populated as null if we cannot populate any values.